src/components/shared/ExternalLink/ExternalLink.tsx

```diff
@@ -1,50 +1,24 @@
-import React, { type ReactNode } from 'react';
-import { type MessageDescriptor, useIntl } from 'react-intl';
+import clsx from 'clsx';
+import React, { type FC } from 'react';
 
-import { type SimpleMessageValues } from '~types/index.ts';
+import { type ExternalLinkProps } from './types.ts';
 
-interface Props {
-  /** Render a react-node into the link */
-  children?: ReactNode;
+const displayName = 'ExternalLink';
 
-  /** The page it should link to */
-  href: string;
-
-  /** A string or a `messageDescriptor` that make up the link's text */
-  text?: MessageDescriptor | string;
-
-  /** Values for text (react-intl interpolation) */
-  textValues?: SimpleMessageValues;
-
-  /*
-   * Allows for link style customization (Eg: we need to disquise the link as a button)
-   * Don't abuse it!
-   */
-  className?: string;
-
-  /*
-   * Show a title over the element, on hover. This is browser native.
-   */
-  title?: string;
-  download?: string | boolean;
-}
-
-const ExternalLink = ({
+const ExternalLink: FC<ExternalLinkProps> = ({
   children,
   href,
-  text,
-  textValues,
   className,
   title,
   download,
-}: Props) => {
-  const { formatMessage } = useIntl();
-  const typeOfText =
-    typeof text == 'string' ? text : text && formatMessage(text, textValues);
-  const linkText = children || typeOfText || href;
+  hasHover,
```

Comment:
In this case, with hasHover flag, I am open to other ideas such as separate styled component etc. :)

Reply:
In my opinion, and If I understand the intent correctly, this component should always have a hover state for accessibility purposes since it's an anchor element

Reply:
thanks for your comment, I removed hasHover totally